package.json

```diff
@@ -7,31 +7,11 @@
-        "build:admin": "pnpm recursive --filter '@comet/admin*' run build",
-        "build:admin:skippable": "test -f packages/admin/admin/lib/index.d.ts && echo 'Skipping Admin build' || $npm_execpath build:admin",
-        "build:cms": "pnpm recursive --filter '@comet/cms*' --filter '@comet/blocks*' run build",
-        "build:cms:admin": "pnpm recursive --filter '@comet/cms-admin' --filter '@comet/blocks-admin' run build",
-        "build:cms:admin:skippable": "test -f packages/admin/cms-admin/lib/index.d.ts && echo 'Skipping CMS Admin build' || $npm_execpath build:cms:admin",
-        "build:cms:api": "pnpm recursive --filter '@comet/cms-api' --filter '@comet/blocks-api' run build",
-        "build:cms:api:skippable": "test -f packages/api/cms-api/lib/index.d.ts && echo 'Skipping CMS API build' || $npm_execpath build:cms:api",
-        "build:cms:site": "pnpm recursive --filter '@comet/cms-site' run build",
-        "build:cms:site:skippable": "test -f packages/site/cms-site/lib/index.d.ts && echo 'Skipping CMS Site build' || $npm_execpath build:cms:site",
-        "build:cms:skippable": "test -f packages/admin/cms-admin/lib/index.d.ts && echo 'Skipping CMS build' || $npm_execpath build:cms",
         "build:storybook": "pnpm recursive --filter '@comet/admin*' --filter '@comet/eslint-plugin' run build && pnpm --filter comet-storybook run build-storybook",
-        "build:lib": "pnpm recursive --filter '@comet/*' run build",
+        "build:packages": "pnpm recursive --filter '@comet/*' run build",
         "build:docs": "pnpm recursive --filter '@comet/eslint-plugin' --filter '@comet/admin*' --filter 'comet-docs' run build",
         "clean": "pnpm recursive run clean",
         "copy-schema-files": "node copy-schema-files.js",
-        "dev:admin": "pnpm recursive --filter '@comet/admin*' run clean && dotenv -c -- dev-pm start @comet-admin",
-        "dev:cms": "pnpm recursive --filter '@comet/cms*' run clean && $npm_execpath build:admin:skippable && dotenv -c -- dev-pm start @cms",
-        "dev:cms:admin": "pnpm recursive --filter '@comet/cms-admin' --filter '@comet/blocks-admin' run clean && $npm_execpath build:admin:skippable && dotenv -c -- dev-pm start @cms-admin",
-        "dev:cms:api": "pnpm recursive --filter '@comet/cms-api' --filter '@comet/blocks-api' run clean && dotenv -c -- dev-pm start @cms-api",
-        "dev:cms:site": "pnpm recursive --filter '@comet/cms-site' run clean && dotenv -c -- dev-pm start @cms-site",
-        "dev:demo": "$npm_execpath build:admin:skippable && $npm_execpath build:cms:skippable && dotenv -c -- dev-pm start @demo",
-        "dev:demo:admin": "$npm_execpath build:admin:skippable && $npm_execpath build:cms:admin:skippable && dotenv -c -- dev-pm start @demo-admin",
-        "dev:demo:api": "$npm_execpath build:cms:api:skippable && dotenv -c -- dev-pm start @demo-api",
-        "dev:demo:site": "$npm_execpath build:cms:site:skippable && dotenv -c -- dev-pm start @demo-site",
```

Comment:
Can we keep these scripts (or some of them like dev:admin, dev:cms, dev:demo) as an alias for the dpm commands?

It's the norm for most projects to have dev scripts (although you could argue that we already break the norm with our install.sh)

Reply:
> It's the norm for most projects to have dev scripts (although you could argue that we already break the norm with our install.sh)

In our projects we only have the `npm run dev` scripts which starts everything: https://github.com/vivid-planet/comet-starter/blob/main/package.json#L5. I wouldn't add that...

> Can we keep these scripts (or some of them like dev:admin, dev:cms, dev:demo) as an alias for the dpm commands?

We could keep some as alias, but I'd prefer if our devs would get used to using dev-pm's groups feature. What do you prefer?

Reply:
Re-added the dev scripts, but put using dev-process-manager directly in a more prominent position. What do you think?